src/pages/docs/deployments/kubernetes/kustomize/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-01-01
+modDate: 2023-01-01
+title: kustomize  
+description: Use kustomize to deploy resources to a Kubernetes cluster.
+navOrder: 9
+---
+
+Octopus supports the deployment of Kubernetes resources through the `kustomize` step.
+
+This step allows you to source your kustomize files from git, perform variable substitutions based on your environment and/or tenant and finally apply the changes to your Kubernetes clusters.
+
+[Kustomize](https://kustomize.io) introduces a template-free way to customize application configuration that simplifies the use of off-the-shelf applications.
+
+## Recommended usages for this step
+
+We list a few different scenarios below to help you figure out what is the best setup for you to deploy your applications to a Kubernetes cluster with Octopus.
+
+*When you use Kustomize for one application configuration*
+
+1. **The kustomize way**  
+   This is the recommended usage if you are already using Kustomize and just want Octopus to orchestrate the deployment.
+   In this scenario, our recommendation is to use `.env` files with Octopus [variable substitution syntax](/docs/projects/variables/variable-substitutions), so we can replace secrets and any other data managed via Octopus variables. These `.env` files are then used by [secretGenerator](https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_secretgenerator_) and/or [configMapGenerator](https://kubectl.docs.kubernetes.io/references/kustomize/builtins/#_configmapgenerator_).
+   Everything else is defined in the `kustomization.yaml` files directly, and overlays should match the same environment structure defined in Octopus itself. 
+
+2. **No overlays**  
```

Comment:
Maybe "A single overlay for Octopus"? 
"No overlays" is confusing because you immediately mentioned that 2 overlays will be defined.